src/test/groovy/org/gradle/profiler/JFRProfilerIntegrationTest.groovy

```diff
@@ -138,32 +139,44 @@ class JFRProfilerIntegrationTest extends AbstractProfilerIntegrationTest {
         latestSupportedGradleVersion  | _
     }
 
+    @Requires({ !OperatingSystem.isWindows() })
```

Comment:
Is it enough to only check the flamegraphs dir on Linux, and run the remainder of the test on Windows as well?

Reply:
Done.